.precaution.ymal.example

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+
+# That way you can say which security linters do you want to skip
+skip_linters: 'bandit'
+
+# Example of Bandit specific options
+# All of them can be found here: https://github.com/PyCQA/bandit#configuration
+bandit:
```

Comment:
I think we should attempt to abstract this more.  If we just blatantly expose bandit config parameters, then why have another file.  Especially if all the settings are the same.